consensus/src/consensus/mod.rs

```diff
@@ -508,6 +514,19 @@ impl ConsensusApi for Consensus {
         self.virtual_processor.import_pruning_point_utxo_set(new_pruning_point, imported_utxo_multiset)
     }
 
+    fn validate_pruning_points(&self) -> ConsensusResult<()> {
+        let pp_info = self.pruning_point_store.read().get().unwrap();
+        if !self.services.pruning_point_manager.is_valid_pruning_point(pp_info.pruning_point) {
```

Comment:
I think we should read `hst` here and pass it to both `is_valid_pruning_point` and `are_pruning_points_in_valid_chain`.
Because (i) they both use it, (ii) the second relies on the chain check performed in the first